Week-3/Session9 Basics JS/divideArrays.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>JavaScript Array Lab</title>
+    <script>
+      let nums = [21, 61, 81, 1];
+      function divideArray(nums) {
+        let evenArr = [];
+        let oddArr = [];
+        for (let i = 0; i < nums.length; i++) {
+          if (nums[i] % 2 == 0) {
+            evenArr.push(nums[i]);
+            evenArr.sort(function (a, b) {
+              return a - b;
+            });
```

Comment:
```suggestion
            evenArr.sort();
```
for natual ascending sort we don't need to specify comparator function